lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -56,6 +56,12 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
     @Input()
     appName: string;
 
+    /** Define which filter id should be selected after reloading. If the filter id doesn't
```

Comment:
remove 
If the filter id doesn't exist or nothing is passed then the first filter will be selected

Reply:
Done.